client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -23,18 +23,18 @@ export default () => {
 
 	const homepageTemplates = templates.filter( template => template.category === 'home' );
 
-	const [ previewedTemplate, setPreviewedTemplate ] = useState< string | null >( null );
+	const [ previewedTemplate, setPreviewedTemplate ] = useState< string | undefined >();
```

Comment:
Could probably just do

```suggestion
	const [ previewedTemplate, setPreviewedTemplate ] = useState< string >( '' );
```

since it's only used in `TemplateSelectorControl` to compare each item's template slug to.

Reply:
I prefer
- `undefined` -> we don't have a slug
- `string` -> it's the slug string

What do you think?

Reply:
Yeah, wfm. My motivation here was mostly to make the type 'look' nicer -- just `string` seemed more succinct than `string | undefined`; under the assumption that we could use `''` as the falsey initial state.
Not feeling too strongly about this, tho, so let's go with `| undefined`.